src/executables/util_cs.c

```diff
@@ -4603,6 +4605,17 @@ memmon (UTIL_FUNCTION_ARG * arg)
       goto error_exit;
     }
 
+  if (finalize_force)
+    {
+      error_code = mmon_disable_force ();
+      if (error_code != NO_ERROR)
+	{
+	  goto error_exit;
+	}
+      fprintf (stdout, msgcat_message (MSGCAT_CATALOG_UTILS, MSGCAT_UTIL_SET_MEMMON, MEMMON_MSG_DISABLE_SUCCESS));
+      goto success_exit;
+    }
+
   /* execute phase */
   error_code = mmon_get_server_info (server_info);
   if (error_code != NO_ERROR)
```

Comment:
I think user need to know which error has occurred during executing utility.

Reply:
Currently, mmon_disable_force() does not fail.